Dockerfile

```diff
@@ -87,53 +87,81 @@ RUN wget https://github.com/jakzal/toolbox/releases/download/v$TOOLBOX_VERSION/t
 # PHP with all extensions needed, correctly configured
 FROM php-base as php-configured
 
-# Extensions .so
+# Install extensions + extension config
 COPY --link --from=ast-builder /usr/local/lib/php/extensions /usr/local/lib/php/extensions
-COPY --link --from=pcov-builder /usr/local/lib/php/extensions /usr/local/lib/php/extensions
-COPY --link --from=extensions-builder /usr/local/lib/php/extensions /usr/local/lib/php/extensions
-
-# Extensions .ini
 COPY --link --from=ast-builder /usr/local/etc/php/conf.d /usr/local/etc/php/conf.d
+COPY --link --from=pcov-builder /usr/local/lib/php/extensions /usr/local/lib/php/extensions
 COPY --link --from=pcov-builder /usr/local/etc/php/conf.d  /usr/local/etc/php/conf.d
+COPY --link --from=extensions-builder /usr/local/lib/php/extensions /usr/local/lib/php/extensions
 COPY --link --from=extensions-builder /usr/local/etc/php/conf.d /usr/local/etc/php/conf.d
 
+# Base php.ini
 COPY --link <<EOF /usr/local/etc/php/conf.d/phpqa.ini
 date.timezone=Europe/London
 memory_limit=-1
 phar.readonly=0
 pcov.enabled=0
 EOF
 
-COPY --link --chmod=755 <<EOF /entrypoint.sh
-#!/usr/bin/env sh
-set -e
+# Validate the PHP configuration
+RUN php -m | grep ast
+RUN php -m | grep pcov
+RUN php -m | grep bcmath
+RUN php -m | grep intl
+RUN php -m | grep zip
+RUN php -m | grep bz2
+RUN php --ini | grep phpqa.ini
 
-(test "\$GITHUB_ACTIONS" = "true" || test "\$CI" = "true") && test -f composer.json && composer install --no-scripts --no-progress
 
-exec "\$@"
-EOF
 ARG TOOLBOX_TARGET_DIR="/tools"
-COPY --link --from=toolbox-downloader --chmod=+x /toolbox.phar ${TOOLBOX_TARGET_DIR}/toolbox
-COPY --link --from=composer:2 /usr/bin/composer ${TOOLBOX_TARGET_DIR}/composer
+WORKDIR ${TOOLBOX_TARGET_DIR}
+ENV PATH="${PATH}:${TOOLBOX_TARGET_DIR}"
 
-# INSTALLATION_DATE is used to bust the docker layer cache
-ARG INSTALLATION_DATE
+# Configure Composer
+COPY --link --from=composer:2 /usr/bin/composer ./composer
+ENV COMPOSER_HOME=${TOOLBOX_TARGET_DIR}/.composer
+ENV PATH="${PATH}:${COMPOSER_HOME}/vendor/bin"
+ENV COMPOSER_ALLOW_SUPERUSER 1
+RUN composer --version
+
+# Configure Toolbox
+COPY --link --from=toolbox-downloader --chmod=+x /toolbox.phar ./toolbox
 ARG TOOLBOX_EXCLUDED_TAGS
-ARG TOOLBOX_VERSION
 ENV TOOLBOX_EXCLUDED_TAGS=${TOOLBOX_EXCLUDED_TAGS}
-ENV TOOLBOX_TARGET_DIR="/tools"
```

Comment:
This should not really be removed I think. It's used by toolbox

Reply:
Fixed in [c5279a5](https://github.com/jakzal/phpqa/pull/371/commits/c5279a5e8332bd00c2f7ce37971903517523a1ac)